parm/config/gefs/config.fcst

```diff
@@ -0,0 +1,366 @@
+#######################################################################
+# COUPLING COMPONENTS
+
+# cpl defaults
+export cpl=".false."
+export cplflx=".false."
+export cplice=".false."
+export cplchm=".false."
+export cplwav=".false."
+
+# cpl changes based on APP
+
+if [[ "${DO_COUPLED}" == "YES" ]]; then
+  export cpl=".true."
+fi
+if [[ "${DO_AERO}" == "YES" ]]; then
+  export cplchm=".true."
+fi
+if [[ "${DO_ICE}" == "YES" ]]; then
+  export cplice=".true."
+  export cplflx=".true."
+fi
+if [[ "${DO_OCN}" == "YES" ]]; then
+  export cplflx=".true."
+fi
+if [[ "${DO_WAVE}" == "YES" ]]; then
+  export cplwav=".true."
+fi
+
+#######################################################################
+
+export FORECASTSH="${HOMEgfs}/scripts/exglobal_forecast.sh"
+#export FORECASTSH="${HOMEgfs}/scripts/exglobal_forecast.py"  # Temp. while this is worked on
+export FCSTEXECDIR="${HOMEgfs}/exec"
+export FCSTEXEC="ufs_model.x"
+
+#######################################################################
+# Model configuration
+export TYPE="nh"
+export MONO="non-mono"
+
+# Use stratosphere h2o physics
+export h2o_phys=".true."
+
+# Options of stratosphere O3 physics reaction coefficients
+export new_o3forc="YES"
+
+export gwd_opt=2
+
+# --GFS.v16 uGWD.v0, used for suite FV3_GFS_v16 and UFS p6 etc
+# do_ugwp=T: use unified CGWD and OGWD, and turbulent orographic form drag (TOFD)
+# do_ugwp=F: use unified CGWD but old OGWD, TOFD is not uded.
+if (( gwd_opt == 1 )); then
+    export knob_ugwp_version=0
+    export do_ugwp=".false."
+    export do_tofd=".false."
+    launch_level=$(echo "${LEVS}/2.35" |bc)
+    export launch_level
+fi
+
+
+# -- uGWD.v1, for suite FV3_GFS_v17 and FV3_GFS_v17p8b etc
+if (( gwd_opt == 2 )); then
+
+    #--used for UFS p7 and p8a
+    #export  knob_ugwp_version=1
+    #export  do_ugwp=".false."
+    #export  do_tofd=".false."
+    #export  do_ugwp_v0=".false."
+    #export  do_ugwp_v1=".true."
+    #export  do_ugwp_v0_orog_only=".false."
+    #export  do_ugwp_v0_nst_only=".false."
+    #export  do_gsl_drag_ls_bl=".true."
+    #export  do_gsl_drag_ss=".true."
+    #export  do_gsl_drag_tofd=".true."
+    #export  do_ugwp_v1_orog_only=".false."
+
+    #--used for UFS p8
+    export knob_ugwp_version=0
+    export do_ugwp=".false."
+    export do_tofd=".false."
+    export do_ugwp_v0=".true."
+    export do_ugwp_v1=".false."
+    export do_ugwp_v0_orog_only=".false."
+    export do_ugwp_v0_nst_only=".false."
+    export do_gsl_drag_ls_bl=".false."
+    export do_gsl_drag_ss=".true."
+    export do_gsl_drag_tofd=".false."
+    export do_ugwp_v1_orog_only=".false."
+    launch_level=$(echo "${LEVS}/2.35" |bc)
+    export launch_level
+fi
+
+# Sponge layer settings
+export tau=10.0
+export rf_cutoff=7.5e2
+export d2_bg_k1=0.20
+export d2_bg_k2=0.04
+export dz_min=6
+export n_sponge=42
+if (( LEVS == 128 )) && [[ "${CDUMP}" =~ "gdas" ]]; then
+   export tau=5.0
+   export rf_cutoff=1.0e3
+   export d2_bg_k1=0.20
+   export d2_bg_k2=0.0
+fi
```

Comment:
Does GEFS need this?

Reply:
I mean lines 144-149

Reply:
These are just copied from GFS for now. They will be pruned later.